### PR TITLE
An article about the Nomad hack in August 2022.

### PR DESCRIPTION
**Pull request by L0nest**

### CODE REVIEW

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]
Harmony Horizon's hack [already exists](https://github.com/1712n/dn-institute/blob/main/content/attacks/posts/2022-06-23-Harmony-Horizon.md). Remove it from your pr.

**Comment by L0nest** [comment]
> Harmony Horizon's hack [already exists](https://github.com/1712n/dn-institute/blob/main/content/attacks/posts/2022-06-23-Harmony-Horizon.md). Remove it from your pr.

File: content/attacks/posts/2022-08-01-Nomad.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- **August 3, 2022 04:05 AM GMT:**[Nomad announced an up to 10% bounty:](https://twitter.com/nomadxyz_/status/1554679735006859264) those who return >90% of funds.
+- **August 5, 2022:** [Nomad published a detailed analysis](https://medium.com/nomad-xyz-blog/nomad-bridge-hack-root-cause-analysis-875ad2e5aacd) of the root causes of the attack.
+
+## Security Failure Causes
+
+**Smart Contract Vulnerability:** The security failure in the Nomad bridge occurred due to a mistake made during the contract update process. This highlights the need for comprehensive code review and change management practices to identify and address vulnerabilities within the smart contract code. Conducting thorough audits can help detect overlapping functionalities or potential vulnerabilities that could be exploited by attackers.
```
Comment (JediFaust) [inline content/attacks/posts/2022-08-01-Nomad.md:31]: Remove last two sentences and add more details about the vulnerability itself

File: content/attacks/posts/2022-08-01-Nomad.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The true identity of the initial attacker remains unknown, as the hack itself was a result of an error. However, it is known that a significant number of accounts exploited this vulnerability. Once other attackers became aware of the situation, they mobilized large numbers of bots to execute similar attacks.
+
+## Losses
+
+After a frenzied hack from hundreds of wallets, the bridge’s TVL dropped from $190,740,000 to $1,794 in mere hours.
+
+## Timeline
+
+- **August 1, 2022 11:25 PM GMT:** [The project team reported](https://twitter.com/nomadxyz_/status/1554246853348036608) that they know about the hack.
```
Comment (JediFaust) [inline content/attacks/posts/2022-08-01-Nomad.md:24]: Rephrase the sentence, fix grammar issue

File: content/attacks/posts/2022-08-01-Nomad.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- **August 2, 2022 10:26 PM GMT:** [The project team stated](https://twitter.com/nomadxyz_/status/1554413278406721537?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1554413278406721537%7Ctwgr%5Ee9a62d942e20b470f1122ce321dad21cece3434b%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fwww.theverge.com%2F2022%2F8%2F2%2F23288785%2Fnomad-bridge-200-million-chaotic-hack-smart-contract-cryptocurrency) that they informed law enforcement authorities and engaged reputable firms specializing in blockchain intelligence and forensics.
```
Comment (JediFaust) [inline content/attacks/posts/2022-08-01-Nomad.md:25]: Fix source link by removing unnecessary characters from URL

File: content/attacks/posts/2022-08-01-Nomad.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+In response to the hack, Nomad took the initiative to offer a bounty of up to 10% for the return of looted funds. Those who returned more than 90% of the stolen funds would be considered "white hat" hackers and would be allowed to keep the remaining amount. Nomad made it clear that no legal action would be taken against individuals who returned the funds. As a result, white hat hackers returned over $36 million to the platform, contributing to the recovery efforts.
+
+## Attackers
+
+The true identity of the initial attacker remains unknown, as the hack itself was a result of an error. However, it is known that a significant number of accounts exploited this vulnerability. Once other attackers became aware of the situation, they mobilized large numbers of bots to execute similar attacks.
+
+## Losses
+
+After a frenzied hack from hundreds of wallets, the bridge’s TVL dropped from $190,740,000 to $1,794 in mere hours.
```
Comment (JediFaust) [inline content/attacks/posts/2022-08-01-Nomad.md:20]: Add a total loss

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2022-08-01-Nomad.md (hunk 1 above)
Comment (L0nest) [reply content/attacks/posts/2022-08-01-Nomad.md]: Corrected

**Review by JediFaust — CHANGES_REQUESTED** [review]
Resolve conflicts with the [main](https://github.com/1712n/dn-institute/tree/main) branch please

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, resolve conflicts with the [main](https://github.com/1712n/dn-institute/tree/main) branch

**Comment by L0nest** [comment]
> Please, resolve conflicts with the [main](https://github.com/1712n/dn-institute/tree/main) branch

Check, please

**Review by JediFaust — APPROVED** [review]

**Comment by JediFaust** [comment]
@evgenydmitriev Can be merged after approval

**Comment by evgenydmitriev** [comment]
image

**Comment by L0nest** [comment]
> image

I'm sorry, I did not come across this check, do I need to make any changes?

**Review by JediFaust — CHANGES_REQUESTED** [review]
Yeah, try to rewrite some parts with your own words

File: content/attacks/posts/2022-08-01-Nomad.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+date: 2022-08-01
+custodians: Nomad
+categories: Bridge Hack
```
Comment (JediFaust) [inline content/attacks/posts/2022-08-01-Nomad.md:4]: Please, reformat headers regarding with [new changes](https://github.com/1712n/dn-institute/tree/main#readme). Here is an example:
```
target-entities: Nomad
entity-types:
- DeFi
- Exchange
attack-types:
- Smart Contract Exploit
```

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by JediFaust — CHANGES_REQUESTED** [review]
I assume misunderstanding happened here, sorry for that. I just pointed before headers as an example, and Nomad is not an Exchange. I wrote previous headers as an example. Here is right headers for Nomad case:
entity-types:
    - DeFi
    - Bridge
attack-types:
    - Smart Contract Exploit
    - Signarute Verification Issue

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, merge [1712n:main](https://github.com/1712n/dn-institute/tree/main) into your branch

**Comment by L0nest** [comment]
> Please, merge [1712n:main](https://github.com/1712n/dn-institute/tree/main) into your branch

Merged

**Review by JediFaust — APPROVED** [review]
Thanks!

**Comment by JediFaust** [comment]
@evgenydmitriev can be merged after approval, all changes addressed

**Review by evgenydmitriev — APPROVED** [review]

**Pull request merged**